clouddriver-api/src/main/java/com/netflix/spinnaker/clouddriver/deploy/ValidationErrors.java

```diff
@@ -33,4 +33,7 @@
   void rejectValue(String field, String errorCode, String defaultMessage);
 
   void rejectValue(String field, String errorCode, Object[] errorArgs, String defaultMessage);
+
+  /** Remove error by error code. */
+  void remove(String errorCode);
```

Comment:
I'm not seeing any use of this aside from in a test - what is the use case here?

Reply:
The idea was we would use it a -nflx DescriptionAuthorizer.  However, I'm trying out the idea of having the DescriptionAuthorizer return an errors object so we don't have to remove an `ObjectError` error.